modules/migration-understanding-data-copy-methods.adoc

```diff
@@ -28,6 +28,12 @@ a|* Slower than the snapshot copy method.
 * Optional data verification significantly reduces performance.
 |===
 
+====
+[NOTE]
+Our Restic and Rsync PV migration assumes that the PVs supported are only volumeMode=filesystem. We *do not* support volumeMode=Block for filesystem migration.
```

Comment:
We usually avoid trying to say "our" or "we", can this be reworded? Also, here are a few suggested updates if you agree they are appropriate.

NOTE: Don't push the "Commit suggestion" button you'll see below this :)

```suggestion
The Restic and Rsync PV migration assumes that the PVs supported are only `volumeMode=filesystem`. Using `volumeMode=Block` for file system migration is _not_ supported.
```